extension/custom.css

```diff
@@ -11,3 +11,38 @@ a[href^="github-mac"] {
 .reponav a[href$="/pulse"] {
 	display: none !important;
 }
+
+/* alternate monospace font stack */
+code,
+kbd,
+pre,
+tt,
+.ace_editor.ace-github-light,
+.blame-sha,
+.blob-code-inner,
+.blob-num,
+.branch-name,
+.commit .sha,
+.commit .sha-block,
+.commit-desc pre,
```

Comment:
Is this necessary with the other `pre` listed above? I would think the `!important` would be enough to enforce the style on all `pre` tags even if they're selected more specifically, unless GitHub is putting `!important` in their CSS (I'd be surprised to see that).

If it's required leave it, of course.

Same comment applies to the couple of other scenarios below.